build.gradle

```diff
@@ -8,11 +9,19 @@ plugins {
 group = 'com.reckure'
 version = '1.0.SNAPSHOT'
 
-sourceCompatibility = JavaVersion.VERSION_11
-targetCompatibility = JavaVersion.VERSION_11
+sourceCompatibility = 11
+targetCompatibility = 11
 
 repositories {
     mavenCentral()
+    maven {
+        name = "ReckueOpenSource"
+        url = "https://maven.pkg.github.com/reckue/commons"
+        credentials {
+            username = 'egnaf'
+            password = '95db80628ed5e1e89e07269982a31ef26c0e69cd'
```

Comment:
Need to create private environment for passwords and any requested secure params.